ptbr/config_cli.py

```
@@ -0,0 +1,842 @@
+
+
+# ============================================================================
+# Public API: load_and_validate_config
+# ============================================================================
+
+
+def load_and_validate_config(
+    file: str | Path,
+    full_or_lora: Literal["full", "lora"] = "full",
+    method: Literal["biencoder", "decoder", "relex", "span", "token"] = "span",
+    validate: bool = True,
```

Ruff (0.15.1)

[warning] Unused function argument: `validate`

(ARG001)